src/main/scala/frontend.scala

```diff
@@ -129,7 +129,7 @@ class HwachaFrontendModule(outer: HwachaFrontend)(implicit p: Parameters) extend
     val ptw = new freechips.rocketchip.rocket.TLBPTWIO()
   })
   val icache = outer.icache.module
-  val tlb = Module(new freechips.rocketchip.rocket.TLB(instruction = true, lgMaxSize = log2Ceil(cacheParams.fetchBytes), TLBConfig(nptlb))(edge, p))
+  val tlb = Module(new freechips.rocketchip.rocket.TLB(instruction = true, lgMaxSize = log2Ceil(cacheParams.fetchBytes), TLBConfig(nSets=1, nWays=nptlb))(edge, p))
```

Comment:
I think we actually want the old number of entries to be sets not ways. This TLB can be quite low performance since each instruction fetched corresponds to so much work.

Reply:
Sure. It's up to you. I can switch it to being sets.